delete property maxPriority (#464)
yiisoft/yii2-queue: UPGRADE.md

This file contains the upgrade notes. These notes highlight changes that could break your
application when you upgrade the package from one version to another.

Upgrade to 3.0.0
----------------

> * The [amqp_interop](docs/guide/driver-amqp-interop.md) has been removed property `maxPriority`. 

Reviewer: ```suggestion
* The `maxPriority` property was removed from [amqp_interop](docs/guide/driver-amqp-interop.md). 
```
